outsettingsvalues/tests.py

```diff
@@ -0,0 +1,82 @@
+"""
+Test outsettingsvalues view.
+"""
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.shortcuts import reverse
+from django.test import Client, RequestFactory, TestCase
+
+from outsettingsvalues.views import BaseView
+
+
+class TestBaseView(TestCase):
+    """
+    BaseView test class.
+    """
+
+    def setUp(self):
+        """
+        Create mane instances for all test.
+
+        :return: None
+        """
+        self.client = Client()
+        self.baseview = BaseView()
+
+    def test_status_code(self):
+        """
+        Test status code about get request of BaseView url without param setting_name.
+
+        :return: True or False
```

Comment:
Seems this function return None, could you clarify docstring or remove the part with return description completely in case, when it trivial?